client/renderSDL/FontChain.cpp

```diff
@@ -116,7 +116,7 @@ size_t FontChain::getGlyphWidthScaled(const char * data) const
 std::vector<FontChain::TextChunk> FontChain::splitTextToChunks(const std::string & data) const
 {
 	// U+FFFD - replacement character (question mark in rhombus)
-	static const std::string replacementCharacter = u8"�";
+	static const std::string replacementCharacter = reinterpret_cast<const char *>(u8"�");
```

Comment:
this character looks as if rendering is incorrect until you read the comment. I suggest to use `\u` literal to make everything clear

looks like we still can't use `u8string` here?

Reply:
Yeah, will do it tomorrow